routes/botSettings.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const botSettings = require('../db/botSettings.js')
+const router = express.Router()
+
+module.exports = router;
+// reset endpoint, use only in testing
+router.get('/start', async (req, res) => {
+    await botSettings.destroy({where: {}})
+    let bot = await botSettings.build({'id': 1,
+                    'botToken': 'Token', 
+                    'emojies':
+                        {'like':'👍',
+                        'dislike':'👎',
+                        'message':'📝',
+                        'report':'‼️'}})
+    bot.save()
+    res.send(200)
+})
+// update data endpoint
+router.put('/change', async (req, res) => {
+    let bot = await botSettings.findByPk(1)
+    bot.botToken = req.body.botToken;
+    bot.emojies = {'like':req.body.like,
+                    'dislike':req.body.dislike,
+                    'message':req.body.message,
+                    'report':req.body.report};
+    await bot.save()
+    res.send(200)
+})
+// get data endpoint
+router.get('/get', async (req, res) => {
+    let bot = await botSettings.findByPk(1)
```

Comment:
Why do you get by pk and hardcode it?

Reply:
Because, its model like Singleton, and we have only one instance of model BotSetting, but maybe I must to use findOne?

Reply:
Got it. That way I would recommend you to put it into the app's configuration. Read more here - https://www.freecodecamp.org/news/how-to-use-node-environment-variables-with-a-dotenv-file-for-node-js-and-npm/